torchdynamo/symbolic_convert.py

```diff
@@ -397,6 +397,24 @@ def call_function(self, fn, args, kwargs):
                     self.push(ConstantVariable(len(item), **options))
                 else:
                     unimplemented(f"`len` with arg type {arg}")
+            elif fn.fn is isinstance:
+                assert not kwargs and len(args) == 2
+                arg, *isinstance_types = args
+                get_type = (
+                    lambda x: x.fn if isinstance(x, BuiltinVariable) else x.as_proxy()
```

Comment:
When would the type be a proxy?  Perhaps we should just require it to be a constant.

Reply:
Perhaps we could simplify this (and above) by adding a `get_type()` method to `VariableTracker` which errors if we don't know the type.

I worry we will by mistake get a proxy type, then create a incorrect `False` node in the graph